test/PrintAsCxx/empty.swift

```diff
@@ -0,0 +1,54 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) %s -typecheck -emit-cxx-header-path %t/empty.h
```

Comment:
Nit: do you need `-typecheck`? Can this just be `%target-swift-frontend -emit-cxx-header-path %t/empty.h`?

Reply:
Yes, as the default mode doesn't type check and in the default mode we get this error as per `canActionEmitClangHeader`:

```
<unknown>:0: error: this mode does not support emitting C++ headers
```